scripts.js

```diff
@@ -0,0 +1,226 @@
+      })
+      .catch(err => console.log(err));
+  }
+
+  insertText(arrQuestion, arrAnswer) {
+    const questionText = this.container.querySelectorAll('.question__title');
+    for (let i = 0; i < arrQuestion.length; i++) {
+      questionText[i].textContent = arrQuestion[i];
+    }
+
+    const answerText = this.container.querySelectorAll('.answer');
+    const valueText = this.container.querySelectorAll('.radio');
+    for (let i = 0; i < arrAnswer.length; i++) {
+      answerText[i].textContent = arrAnswer[i];
+      valueText[i].value = arrAnswer[i];
+    }
+  }
+
+  assignHandlerAnswer() {
+    Array.from(this.container.querySelectorAll('.question .question__answers'))
+      .forEach(answers => {
+        answers.addEventListener("click", this.checkResultBind);
+      })
+  }
+
+  assignHandlerButton() {
+    this.start.addEventListener("click", this.startGame.bind(this));
+    this.next.addEventListener("click", this.switchNextQuestion.bind(this));
+    this.previous.addEventListener("click", this.switchPreviousQuestion.bind(this));
+    this.buttonName.addEventListener("click", this.eventButtonName.bind(this));
+    this.restart.addEventListener("click", this.eventRestart.bind(this));
+  }
+
+  checkResult(event) {
+    let target = event.target;
+    if (target.tagName != 'LABEL') return;
+
+    this.disableButton();
+
+    let answer = target.previousElementSibling.value;
+    let isCorrect = this.correctDefaultAnswers.includes(answer) || this.correctRequestAnswers.includes(answer);
+    if (isCorrect) {
+      target.style.background = "green";
+      this.countAnswers++;
+    } else {
+      target.style.background = "red";
+    }
+
+    clearTimeout(this.timerShowQuestion);
+
+    this.timerCheckQuestion = setTimeout(this.switchNextQuestion.bind(this), 1500);
+  }
+
+  disableButton() {
+    const disableBtn = this.slides[this.countSlide].querySelectorAll('.radio');
+    Array.from(disableBtn).forEach(item => item.setAttribute("disabled", "disabled"));
+
+    const disableAnswers = this.slides[this.countSlide].querySelector('.question__answers');
+    disableAnswers.removeEventListener("click", this.checkResultBind);
+  }
+
+  startGame() {
+    this.start.classList.add('none');
+    this.next.classList.remove('none');
+    this.previous.classList.remove('none');
+    this.switchQuestion(this.countSlide);
+    console.log(`start ${this.countSlide + 1}`);
+  }
+
+  switchQuestion(switchedSlide) {
+    this.slides[this.countSlide].classList.remove('active');
+    this.slides[switchedSlide].classList.add('active');
+
+    if (this.timerShowQuestion !== null) clearTimeout(this.timerShowQuestion);
+    if (this.countSlide !== 3) {
+      this.timerShowQuestion = setTimeout(() => {
+        this.disableButton();
+        this.switchNextQuestion();
+      }, 10000);
+    }
+  }
+
+  switchNextQuestion() {
+    if (this.countSlide === 2) {
+      this.next.textContent = 'check';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/game-js/pullRequest?prid=7705696)
```suggestion
      this.next.textContent = "check";
```